plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -0,0 +1,197 @@
+package io.jenkins.plugins.casc;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class SchemaGeneration {
+
+    final static JSONObject schemaTemplateObject = new JSONObject()
+        .put("$schema", "http://json-schema.org/draft-07/schema#")
+        .put("description", "Jenkins Configuration as Code")
+        .put("type", "object");
+
+    public static JSONObject generateSchema() {
+
+        /**
+         * The initial template for the JSON Schema
+         */
+        JSONObject schemaObject = new JSONObject(schemaTemplateObject.toString());
```

Comment:
@timja I'm kind of sure that Intellij asked me to import `org.everit.json` while constructing JSONObject

Reply:
It doesn’t appear to be used though?

Reply:
It does becasue if we remove it `JSONObject`  then requires an import